CsCore/xUnitTests/src/Plugins/CsCoreXUnitTests/com/csutil/tests/http/CsvParserTests.cs

```diff
@@ -56,6 +57,66 @@ private class Person {
             public string Location { get; set; }
         }
 
-    }
+        [Fact]
+        public async Task ExampleUsage3() {
+            // Made ConvertToJson Test based on ´ExampleUsage1´
+
+            string sampleCsvData = "Name,Age,Location\nAlice,25,New York\nBob,30,San Francisco\n";
+            using var stream = new MemoryStream(Encoding.Default.GetBytes(sampleCsvData));
+            List<List<string>> parsedData = CsvParser.ReadCsvStream(stream);
+
+            var dataAsJson = CsvParser.ConvertCsvToJArray(parsedData);
+            
+            Assert.Equal("Alice", dataAsJson[0]["Name"].ToString());
+            Assert.Equal("25", dataAsJson[0]["Age"].ToString());
+            Assert.Equal("New York", dataAsJson[0]["Location"].ToString());
+
+            Assert.Equal("Bob", dataAsJson[1]["Name"].ToString());
+            Assert.Equal("30", dataAsJson[1]["Age"].ToString());
+            Assert.Equal("San Francisco", dataAsJson[1]["Location"].ToString());
+
+        }
+
+        [Fact]
+        public async Task ExampleUsage4() {
+            string sampleCsvData = "Name,Age,Location\nAlice,25,New York\nBob,30,San Francisco\n";
+
+            // Creating a MemoryStream from the CSV data
+            using var stream = new MemoryStream(Encoding.Default.GetBytes(sampleCsvData));
+
+            // Converting CSV data to a JSON array using CsvParser
+            var dataAsJson = CsvParser.ReadCsvStreamAsJson(stream);
+            
+
+            // Creating a List of MyUsers
+            List<MyUser> users = new List<MyUser>(dataAsJson.ToObject<List<MyUser>>());
```

Comment:
Lets put these 2 methods together into a 
`List<MyUser> users = CsvParser.ReadCsvStreamAs<MyUser>(stream);`

Inside this `public List<T> ReadCsvStreamAs<T>(stream)` method please use the JsonReader wrapper class so that a developer can decide themselves if they use Newtonsoft for the json parsing (which is the default implementation that JsonReader does) or if they use something else internally.